public/custom.css

```diff
@@ -609,3 +609,8 @@ textarea.form-control[readonly] {
   background-color: inherit;
   color: inherit;
 }
+
+.show-logs-button {
```

Comment:
This is ugly. Not sure how to make it nice though.

Reply:
Agreed. Let's wait for @jgiardino's design input before fixing it, though.